packages/shared/lib/typings/bridge.ts

```diff
@@ -149,7 +150,6 @@ export type MessageResponse =
     | SetAliasResponse
     | DeleteStorageResponse
     | LockStrongholdResponse
-    | StrongholdStatusResponse
```

Comment:
This line was removed because it was a duplicate. See [line #146](https://github.com/iotaledger/firefly/blob/ce9076f8d4efdb346ff8733ecd5619e12069c067/packages/shared/lib/typings/bridge.ts#L146)